Nodejs/Product/Nodejs/SharedProject/CommonProjectNode.cs

```diff
@@ -528,6 +466,12 @@ protected override void Dispose(bool disposing)
                     this.idleManager.OnIdle -= this.OnIdle;
                     this.idleManager.Dispose();
                 }
+
+                if (this.fileWatcher != null)
+                {
+                    this.fileWatcher.FolderChangedOnDisk -= this.FolderChangedOnDisk;
```

Comment:
>this.fileWatcher.FolderChangedOnDisk -= this.FolderChangedOnDisk; [](start = 20, length = 65)

The other dispose call doesn't appear to unsubscribe first.  Is the difference important?

Reply:
Looks like `Close` also calls `Dispose` so removing there